backend/Dockerfile

```diff
@@ -39,10 +39,17 @@ RUN export arch_val="$(dpkg --print-architecture)"; \
 RUN pip install yapf==0.16.2 flake8==3.3.0 requests==2.19.1
 
 # Install Docker (client only)
-ENV DOCKERVERSION=17.03.2-ce
-RUN curl -fsSLO https://download.docker.com/linux/static/stable/x86_64/docker-${DOCKERVERSION}.tgz && \
-    tar xzvf docker-${DOCKERVERSION}.tgz --strip 1 -C /usr/local/bin docker/docker && \
-    rm docker-${DOCKERVERSION}.tgz
+ENV DOCKERVERSION=17.09.1-ce
+RUN export arch_val="$(dpkg --print-architecture)"; \
+    if [ "$arch_val" = "arm64" ]; then \
+        curl -fsSLO https://download.docker.com/linux/static/stable/aarch64/docker-${DOCKERVERSION}.tgz && \
+        tar xzvf docker-${DOCKERVERSION}.tgz --strip 1 -C /usr/local/bin docker/docker && \
+        rm docker-${DOCKERVERSION}.tgz; \
+    else \
```

Comment:
Can we special-case amd64 too, so if you build on something else you get an error and you know you have to fix it?

Reply:
Sure, will update the PR. BTW, I had to bump the Docker client version a bit since the previous used version was not built for ARM.